### DISCUSSION TITLE
CLI usage - stdout

### DISCUSSION THREAD

**Discussion by CrysoK** [kevinzg/facebook-scraper#489] [Q&A] [answered]
Is it possible to choose stdout as output instead of a .csv file?
I'm trying to use the program from Node.js.

**Comment by neon-ninja** [comment, marked as answer]
Give https://github.com/kevinzg/facebook-scraper/commit/f1383b45301f39ff36464e1d30ac2209f56912f1 a try. With this commit, passing `--filename -` or `-f -` will output to stdout

**Reply by CrysoK** [reply]
It works. Although I think I will try to find a way to run `get_posts` from Node.js.
